src/test/java/ee/buerokratt/ruuter/domain/steps/http/HttpPostStepTest.java

```diff
@@ -108,6 +120,13 @@ void execute_shouldExecuteDefaultActionWhenRequestIsInvalid(WireMockRuntimeInfo
         when(defaultAction.getBody()).thenReturn(new HashMap<>());
         when(defaultAction.getQuery()).thenReturn(new HashMap<>());
 
+        when(properties.getDefaultServiceInCaseOfException()).thenReturn(httpDefaultService);
+        doCallRealMethod().when(httpDefaultService).executeHttpDefaultAction(eq(ci), anyString());
+        when(ci.getRequestOrigin()).thenReturn("");
+        when(properties.getHttpCodesAllowList()).thenReturn(new ArrayList<>() {{add(200);}});
+        when(httpResponse.body()).thenReturn("body");
```

Comment:
You dont need to mock these `httpResponse` values, because they already exist on the ResponseEntity you created on line 110